recipes/graphaligner/meta.yaml

```diff
@@ -1,13 +1,13 @@
 {% set name = "GraphAligner" %}
-{% set version = "1.0.11" %}
+{% set version = "1.0.12" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://github.com/maickrau/{{ name }}/files/4498823/{{ name }}.tar.gz
-  sha256: 815d5f4df0d114917740387ba5e7fd5304ff6866beacef37fda851016c1b8c91
+  url: https://github.com/maickrau/{{ name }}/files/4870614/{{ name }}.tar.gz
```

Comment:
Why don't you want to use release source that BioCondaBot can auto bump for future releases?

`https://github.com/maickrau/{{name}}/archive/v{{ version }}.tar.gz`

Reply:
Because the tarball generated by github doesn't include git submodules and so won't compile